Marlin/src/core/boards.h

```diff
@@ -366,21 +366,22 @@
 #define BOARD_BTT_SKR_PRO_V1_1        4207  // BigTreeTech SKR Pro v1.1 (STM32F407ZG)
 #define BOARD_BTT_SKR_PRO_V1_2        4208  // BigTreeTech SKR Pro v1.2 (STM32F407ZG)
 #define BOARD_BTT_BTT002_V1_0         4209  // BigTreeTech BTT002 v1.0 (STM32F407VG)
-#define BOARD_BTT_GTR_V1_0            4210  // BigTreeTech GTR v1.0 (STM32F407IGT)
-#define BOARD_LERDGE_K                4211  // Lerdge K (STM32F407ZG)
-#define BOARD_LERDGE_S                4212  // Lerdge S (STM32F407VE)
-#define BOARD_LERDGE_X                4213  // Lerdge X (STM32F407VE)
-#define BOARD_VAKE403D                4214  // VAkE 403D (STM32F446VET6)
-#define BOARD_FYSETC_S6               4215  // FYSETC S6 (STM32F446VET6)
-#define BOARD_FYSETC_S6_V2_0          4216  // FYSETC S6 v2.0 (STM32F446VET6)
-#define BOARD_FYSETC_SPIDER           4217  // FYSETC Spider (STM32F446VET6)
-#define BOARD_FLYF407ZG               4218  // FLYF407ZG (STM32F407ZG)
-#define BOARD_MKS_ROBIN2              4219  // MKS_ROBIN2 (STM32F407ZE)
-#define BOARD_MKS_ROBIN_PRO_V2        4220  // MKS Robin Pro V2 (STM32F407VE)
-#define BOARD_MKS_ROBIN_NANO_V3       4221  // MKS Robin Nano V3 (STM32F407VG)
-#define BOARD_ANET_ET4                4222  // ANET ET4 V1.x (STM32F407VGT6)
-#define BOARD_ANET_ET4P               4223  // ANET ET4P V1.x (STM32F407VGT6)
-#define BOARD_FYSETC_CHEETAH_V20      4224  // FYSETC Cheetah V2.0
+#define BOARD_BTT_E3_RRF              4210  // BigTreeTech GTR v1.0 (STM32F407IGT)
```

Comment:
Comment should be updated to
`#define BOARD_BTT_E3_RRF 4210 // BigTreeTech E3 RRF (STM32F407IGT)`

Reply:
STM32F4075VX